src/time/TimeOfInterest.ts

```diff
@@ -1,18 +1,20 @@
+import { Time } from './timeTypes';
 import Location from '../earth/Location';
-import {timeCalc} from '../utils';
-import {Time} from './timeTypes';
+import { timeCalc } from '../utils';
 
 export default class TimeOfInterest {
-    public jd: number = 0.0;
-    public T: number = 0.0;
 
-    constructor(public time: Time) {
-        this.jd = timeCalc.time2julianDay(time);
-        this.T = timeCalc.julianDay2julianCenturiesJ2000(this.jd);
+    private readonly julianDay: number;
```

Comment:
As these private variables are only used in the TOI class, I think it is not important to give them a full meaningful full name. Important are the names of the methods.

Reply:
ok.